src/aquarium-optimized/dawn/BufferDawn.cpp

```diff
@@ -21,10 +21,15 @@ BufferDawn::BufferDawn(ContextDawn *context,
       mStride(0),
       mOffset(nullptr) {
   mSize = numComponents * sizeof(float);
+
   // Create buffer for vertex buffer. Because float is multiple of 4 bytes,
   // dummy padding isnt' needed.
   int bufferSize = sizeof(float) * static_cast<int>(buffer->size());
-  mBuf = context->createBuffer(bufferSize, mUsage | wgpu::BufferUsage::CopyDst);
+  wgpu::BufferDescriptor descriptor;
+  descriptor.usage = mUsage | wgpu::BufferUsage::CopyDst;
+  descriptor.size = bufferSize;
+  descriptor.mappedAtCreation = false;
```

Comment:
The default value of mappedAtCreation is false, so it's OK to skip setting it explicitly.

Reply:
Oh it's a small issue and shouldn't block us from merging this PR.